alpine/docker-entrypoint.sh

```diff
@@ -7,9 +7,17 @@ if [[ "$1" == "kong" ]]; then
   PREFIX=${KONG_PREFIX:=/usr/local/kong}
 
   if [[ "$2" == "docker-start" ]]; then
-    kong prepare -p "$PREFIX"
+    chown kong "$PREFIX"
+    su-exec kong kong prepare -p "$PREFIX"
+    
+    if [ ! -z ${KONG_STREAM_LISTEN+x} ]; then
+      setcap cap_net_raw=+ep /usr/local/openresty/nginx/sbin/nginx
+    fi
 
-    exec /usr/local/openresty/nginx/sbin/nginx \
+    chmod 777 /proc/self/fd/1
```

Comment:
This `chmod` is really weird. I think I remember watching you run into this issue, but it really shouldn't happen in the first place as far as I know...